ejabberd/ejabberd.spec

```diff
@@ -76,25 +76,25 @@ Provides:          %{name} = %{version}-%{release}
 
 ###############################################################################
 
-%description 
-ejabberd is an open source Jabber/XMPP server designed from the ground up to 
+%description
+ejabberd is an open source Jabber/XMPP server designed from the ground up to
 be the building bricks of highly critical messaging systems.
 
-Written in Erlang programming language, ejabberd is cross-platform, 
-fault-tolerant, clusterable, very modular and highly versatile. It can be 
+Written in Erlang programming language, ejabberd is cross-platform,
+fault-tolerant, clusterable, very modular and highly versatile. It can be
 extended in other programming languages, such as Elixir.
 
-Designed to be massively scalable, it is widely used to power web scale 
-deployments across many software industries: Mobile messaging, Social 
-Networks, Gaming, Internet of Things, 
+Designed to be massively scalable, it is widely used to power web scale
+deployments across many software industries: Mobile messaging, Social
+Networks, Gaming, Internet of Things,
 
-ejabberd is taking great care of XMPP compliance, implementing most of the 
+ejabberd is taking great care of XMPP compliance, implementing most of the
 XMPP extensions published by the XMPP Standard Foundation.
 
-To innovate even further, the core development team is constantly working 
+To innovate even further, the core development team is constantly working
 with other open source communities to create bridges and elegant features.
 
-This ejabberd community site is a hub for all people that are interested in 
```

Comment:
Don't remove spaces in description, it's not problem, it's real spaces between words in description.